### PR TITLE
4.0.32 release notes

### PR DESCRIPTION
**Pull request by GormFrank**

### CODE REVIEW

**Review by crazybat — APPROVED** [review]

File: site/pages/docs/versions/dwnld-en.hbs [hunk 1]
```diff
@@ -163,28 +187,40 @@
...
+				<tr>
+					<td>Web Experience Toolkit (WET)</td>
+					<td><a href="https://github.com/wet-boew/wet-boew/archive/v4.0.32.zip">Download<span class="wb-inv"> - v4.0.32 source code - Web Experience Toolkit (WET) theme</span></a></td>
+					<td><a href="https://github.com/wet-boew/wet-boew/releases/download/v4.0.32/wet-boew-dist-4.0.32.zip">Download<span class="wb-inv"> - v4.0.32 production files - Web Experience Toolkit (WET) theme</span></a></td>
+				</tr>
+			</tbody>
+		</table>
+		<h5>Browser supported:</h5>
+		<ul>
+			<li>Chrome 75</li>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-en.hbs]: The list of supported browser need to be updated.

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 2]
```diff
@@ -168,29 +192,40 @@
...
+		<h5>Fureteur supporté:</h5>
+		<ul>
+			<li>Chrome 75</li>
+			<li>Chrome 74</li>
+			<li>Safari 12.1.1</li>
+			<li>Firefox 67.0.3</li>
+			<li>Firefox 66.0.5</li>
+			<li>Firefox ESR - 60.7.1</li>
+			<li>IE 11</li>
+			<li>Edge 44</li>
```
Comment (duboisp) [inline site/pages/docs/versions/dwnld-fr.hbs]: List of supported browser need to be updated

File: site/pages/docs/versions/v4.0/v4.0.32-en.hbs [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+		<ul>
+			<li>Chrome 75</li>
+			<li>Chrome 74</li>
+			<li>Safari 12.1.1</li>
+			<li>Firefox 67.0.3</li>
+			<li>Firefox 66.0.5</li>
+			<li>Firefox ESR - 60.7.1</li>
+			<li>IE 11</li>
+			<li>Edge 44</li>
+		</ul>
```
Comment (duboisp) [inline site/pages/docs/versions/v4.0/v4.0.32-en.hbs]: List of supported browser need to be updated

File: site/pages/docs/versions/v4.0/v4.0.32-fr.hbs [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+		<p>Tel que défini par la <a href="https://wet-boew.github.io/wet-boew-documentation/decision/2.html" hreflang="en">Décision de conception 2&nbsp;: Fureteur supporté</a> (en anglais)</p>
+		<ul>
+			<li>Chrome 75</li>
+			<li>Chrome 74</li>
+			<li>Safari 12.1.1</li>
+			<li>Firefox 67.0.3</li>
+			<li>Firefox 66.0.5</li>
+			<li>Firefox ESR - 60.7.1</li>
+			<li>IE 11</li>
+			<li>Edge 44</li>
```
Comment (duboisp) [inline site/pages/docs/versions/v4.0/v4.0.32-fr.hbs]: List of supported browser need to be updated

**Review by duboisp — CHANGES_REQUESTED** [review]
I am going to do the request changes I asked

**Review by duboisp — APPROVED** [review]

**Review by duboisp — APPROVED** [review]

**Pull request merged**